apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SimpleSubscriberTest.cs

```diff
@@ -489,24 +508,6 @@ public void OnlyOneStart()
             }
         }
 
-        [Fact]
```

Comment:
Why was this test removed?

Reply:
It's now OK to call StopAsync() multiple times.